### ISSUE TITLE
Multiple contacts with same surname - bug

### ISSUE THREAD

**Issue by fbentele** [dbrgn/pyxtra#23]
Say, we have lots of "Simon" 's in our contact list. When I autocomplete Sim->tab I get Simon but cant select one (after space it'll add a new receicer). Know what I mean?

**Comment by dbrgn** [comment]
Yep. Use the current git version, it should be fixed there.

If not, well, that would suck :)

**Comment by dbrgn** [comment]
Oh, and please report whether that worked or not... Thanks.

**Comment by fbentele** [comment]
Sorry, it didn't... I get still the same problem

**Comment by dbrgn** [comment]
Fffffuuuu... :)

It works for me (Linux). What environment are you using?

Commit dbbc9648be39143f5ab1f80c615e023c95b0d3d4 solved the problem for me.

**Comment by fbentele** [comment]
Mac OSX 10.7.2 (Lion). I can try a complete re-install, this might help...

**Comment by dbrgn** [comment]
Would be cool, yep. As far as i know, it works for @petermanser... right?

**Comment by fbentele** [comment]
Nope, sorry didn't work out, am I missing something? Config maybe?

**Comment by dbrgn** [comment]
No, it should not depend on the configuration. It has to do with the OS X readline implementation.

Maybe @petermanser knows more.

**Comment by petermanser** [comment]
k. not sure if I got you right.

As space is a seperator for contacts, it searches for new contacts... seems to be a bug only on Mac OS X.

Solution for you: Try typing "Simon" instead of "Simon " and tab, so it shouldn't cause a problem.
